### PR TITLE
Use phantomjs2

### PR DESCRIPTION
**Pull request by waghanza**
`Trusty` environment on `travis` has **phantomjs** `2.0.0` instead of `1.9.3` in `precise`

@see https://github.com/travis-ci/travis-ci/issues/3225#issuecomment-151367517

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -29,6 +29,8 @@ gemfile:
 
 services:
   - mongodb
+  - mysql
+  - postgresql
```
Comment (mshibuya) [inline .travis.yml]: Looks unrelated to PhantomJS, is this correct?

Comment (waghanza) [inline .travis.yml]: nop, in fact on travis  `precise` **env** (default) those services are loaded by default, but with `trusty` env (to have phantom2) we must list all service that have to be started

File: .travis.yml [hunk 2]
```diff
@@ -1,11 +1,8 @@
-dist: trusty
-
```
Comment (mshibuya) [inline .travis.yml]: You want to switch to `trusty`, why is removed?

Comment (waghanza) [inline .travis.yml]: You're write. Kind a mess

**Comment by waghanza** [comment]
Closed, and re-opened #2873
